labgrid/util/proxy.py

```diff
@@ -1,21 +1,35 @@
+import os
+
+from urllib.parse import urlsplit, urlunsplit
+
 from .ssh import sshmanager
 
 from ..resource.common import NetworkResource, Resource
```

Comment:
`NetworkResource` is not used anymore, remove the import.

Reply:
OK.